x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/slm/SnapshotRetentionTaskTests.java

```diff
@@ -239,154 +223,6 @@ private void retentionTaskTest(final boolean deletionSuccess) throws Exception {
         }
     }
 
-    public void testSuccessfulTimeBoundedDeletion() throws Exception {
-        timeBoundedDeletion(true);
-    }
-
-    public void testFailureTimeBoundedDeletion() throws Exception {
-        timeBoundedDeletion(false);
-    }
-
-    private void timeBoundedDeletion(final boolean deletionSuccess) throws Exception {
```

Comment:
These tests seemed irrelevant with the removal of the time bound on delete runs.